src/app/shared/model/charging-stations.model.ts

```diff
@@ -0,0 +1,31 @@
+import { KeyValue } from 'types/GlobalType';
+import { Utils } from 'utils/Utils';
+
+export const CONNECTORS: KeyValue[] = [
```

Comment:
You can automate this with an new Array(26)...

Reply:
Done.